third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/TensorMemoryToLLVM.cpp

```diff
@@ -236,7 +236,7 @@ LinearLayout getTileLayout(MLIRContext *ctx, TMemAccessAtom atom, int bitwidth,
   bases[kWarp].push_back({64, 0});
   auto ret = LinearLayout(bases, {{kRow, 128}, {kCol, nCol}}, false);
   // Broadcast the row dimension if it's smaller than 128
-  ret = ensureLayoutNotLargerThan(ret, {{kRow, nRow}, {kCol, nCol}}, true);
+  ret = ensureLayoutNotLargerThan(ret, {{kRow, nRow}, {kCol, nCol}});
```

Comment:
The default dim is set to "". Then this is different from the original behavior.
But again, I'm wondering if there is a case that we want to remove a dim other than the "register".

Reply:
The default was `true`, we just passed it explicitly here, and meant that we keep broadcasting bases. Which is now equivalent to passing an empty StringRef.